config/default.yml

```diff
@@ -154,6 +154,12 @@ PreCommit:
       - '**/Makefile'
       - '.gitmodules'
 
+  HtmlTidy:
```

Comment:
What sort of use does this tool have, and how does it relate to the `W3cHtml` pre-commit hook?

I'm tempted say this should be opt-in.